components.js

```diff
@@ -550,6 +550,11 @@ var components = {
 			"require": "markup",
 			"owner": "Golmote"
 		},
+		"xojo": {
+			"title": "Xojo (REALbasic)",
+			"require": "basic",
```

Comment:
I changed my mind in the middle, and forgot to remove it here, indeed. Thank you so much for reviewing ;)